discounts/rust/order-discounts/fixed-amount/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
```

Comment:
I believe the other functions in this directory have a `Cargo.toml.liquid` instead of a `Cargo.toml`. Do we want that?

Reply:
Is that because the CLI populates those and creates `Cargo.toml` for you from the default template? I was using `Cargo.toml` so I could actually compile and run the tests lol

Reply:
:100: it makes it way harder to :tophat: in this repo, but surely there is a reason we introduced it in #65 (CC @nickwesselman)

Reply:
Yeah we need `Cargo.toml` to be dynamic so that the [package name](https://github.com/Shopify/function-examples/blob/main/discounts/rust/order-discounts/default/Cargo.toml.liquid#L2) can match the configured [binary path](https://github.com/Shopify/function-examples/blob/main/discounts/rust/order-discounts/default/shopify.function.extension.toml.liquid#L7) for the CLI.

I didn't realize y'all 🎩 'd directly from the repo. Maybe [this script](https://github.com/Shopify/function-examples/blob/347609abf16889f0d10c2c258949e66d39b4f9c6/util/expand-liquid.js) from my abandoned PR would be helpful? The idea is to expand the liquid templates in-place and then `.gitignore` the results.

Reply:
> I didn't realize y'all 🎩 'd directly from the repo.

I don't usually push them but with `Cargo.toml` I get the compiler errors and run the tests in VS Code. Without it I don't know if I've written broken Rust code 😆